apps/marketplace/constants/messageConstants.js

```diff
@@ -11,8 +13,17 @@ export const ACCOUNT_TAKEN =
   'An account with this email domain already exists. Someone in your team may have already created an account with the Marketplace.'
 export const UNABLE_TO_RESET = 'The Digital Marketplace encountered an error trying to reset your password.'
 export const UNABLE_TO_SEND = 'The Digital Marketplace encountered an error trying to send the reset password email.'
-export const LOGIN_FAILED =
-  "Make sure you've entered the right email address and password. Accounts are locked after 5 failed attempts."
+export const LOGIN_FAILED = (
+  <React.Fragment>
+    <li>
+      Make sure you&apos;ve entered the right email address and password. Accounts are locked after 5 failed attempts.
+    </li>
+    <li>
+      Please <a href="https://marketplace1.zendesk.com/hc/en-gb/articles/360001050936">contact our support team </a>
+      if you need to unlock your account
+    </li>
+  </React.Fragment>
+)
```

Comment:
Do we need to update the following files?  We could import `rootPath` in case that changes in the future (eg. `${rootPath}/login`).
apps/marketplace/components/Header/UnauthenticatedHeaderActions.js
apps/marketplace/components/Header/Mobile/UnauthenticatedMobileLinks.js
apps/marketplace/components/Onboarding/SellerOnboarding.js
apps/marketplace/components/ResetPassword/ResetPasswordForm.js
src/bundles/Header/Header.js

Reply:
For the following above files add:
- import rootPath
`import { rootPath } from '../../../routes'`

-then add the rootPath link
`<a href={`${rootPath}/login`}>Sign in</a>`

- run test
`run npm run test`

You should get something similar to the following"
FAIL  apps/***********/components/Onboarding/UserOnboarding.test.js
  ● Test suite failed to run

    TypeError: require.context is not a function

```
       5 | // no require.context in tests
       6 | // https://github.com/facebookincubator/create-react-app/issues/517#issuecomment-243488178
    >  7 | const fileList = require.context('../icons', true, /[\s\S]*$/)
         |                          ^
       8 | 
       9 | fileList.keys().forEach(x => {
      10 |   const file = x.replace('./', '')
```

      at Object.context (apps/shared/Icon/_getIcons.js:7:26)
      at Object.<anonymous> (apps/shared/Icon/Icon.js:5:1)
      at Object.<anonymous> (apps/shared/Icon/index.js:1:1)
      at Object.<anonymous> (apps/***********/components/Feedback/Feedback.js:6:1)
      at Object.<anonymous> (apps/***********/components/Brief/BriefSubmitted.js:3:1)
      at Object.<anonymous> (apps/***********/pages/BriefPage.js:12:1)
      at Object.<anonymous> (apps/***********/routes.js:9:1)
      at Object.<anonymous> (apps/***********/components/Onboarding/SellerOnboarding.js:2:1)
      at Object.<anonymous> (apps/***********/components/Onboarding/UserOnboarding.test.js:4:1)

Found these links to be helpful:

- https://stackoverflow.com/questions/38332094/how-can-i-mock-webpacks-require-context-in-jest
- https://github.com/storybookjs/storybook/issues/2487

Reply:
Looks like we needed to add `jest.mock('shared/Icon/_getIcons')` to a few test files.